jep/0000/README.adoc

```diff
@@ -0,0 +1,316 @@
+solves a similar problem but is not suitable here.
+On the one hand, this JEP involves deploying from `master` (or perhaps another trusted origin branch),
+so there is no need for the precautions used in JEP-305 to check that the deployed bits match expected metadata,
+or the split between CI build and deployment needed to guard a single Artifactory token from malicious (especially forked) PRs.
+And on the flip side, the requirement for a secure execution environment is more stringent:
+if ci.jenkins.io were to be compromised, malicious binaries could be deployed to the user-facing update center,
+not merely an experimental repository used mostly by other CI builds for prerelease testing.
+
+=== Frequency of deployments
+
+The whole point of this JEP is to encourage automatic and frequent deployments.
+If it is widely adopted, there are some risks to this frequency.
+(These are not blockers to experimentation on a few repositories.)
+
+Artifactory might not be able to handle the traffic.
+This is already a concern generally with our hosted Artifactory,
+but the Jenkins project is looking into what precisely the limits are.
+
+Jenkins administrators might tire of constantly seeing entries in the *Plugin Manager » Updates* tab.
+In many cases, there may be few or no behavioral changes in a release, just code cleanups or POM tidying.
+While having these releases is sometimes valuable for PCT, they are not valuable to administrators.
+One mitigation is to slow down the frequency at which the update center is automatically checked, currently one day.
+Another option is to display release notes in the plugin manager GUI so that “chore” updates are more easily ignored.
+
+Releases of development-time components (`plugin-pom`, `bom`, `jenkins-test-harness`, etc.)
+or of widely used API plugins (`workflow-step-api-plugin`, `credentials-plugin`, etc.)
+might create “Dependabot storms” whereby one minor change in a base component triggers a release,
+followed by PRs to intermediate-level components which are then merged and trigger releases,
+followed by PRs to higher-level components with their own releases.
+Excessive updates could consume a lot of CI time and exacerbate the previously mentioned risks.
+
+For any such issues, or for maintainers who prefer to do manual sanity checks prior to release rather than when merging PRs,
+there is another option:
+link:https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/[manual triggers]
+can be used to deploy from a given branch on demand, rather than automatically upon push.
+This is also likely to be the preferred trigger for backport branches.
+Compared to running MRP locally, this is still much less effort for maintainers,
+though such a trigger ought to be sure to validate that there is a passing Jenkins CI check before proceeding.
```

Comment:
The latter, I think.

I do not really want the Action to be running tests, because that is just another environment to maintain, and Dockerized tests will be quietly skipped.